recipes/paraview/v1/gh200/environments.yaml

```diff
@@ -0,0 +1,63 @@
+nvidia:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+      - toolchain: llvm
+        spec: nvhpc
+  unify: when_possible
+  mpi:
+      spec: cray-mpich@8.1.29%nvhpc
+      gpu: cuda
+  specs:
+  - boost%gcc ~mpi
+  - python@3.10%gcc
+  #- eccodes@2.25.0%nvhpc +tools +fortran +aec +openmp jp2k=jasper
+  - cmake%gcc
+  - cuda@12.3%gcc
+  - hdf5%gcc
+  - hwloc%gcc
+  - netcdf-c%gcc
+  - netcdf-cxx4%gcc
+  - netcdf-fortran%nvhpc
+  - numactl%gcc
+  - osu-micro-benchmarks@5.9%nvhpc
+  # everything needed for nccl on SS11
+  - aws-ofi-nccl@master%gcc
+  - nccl%gcc
+  - nccl-tests%gcc
+  # The following are required to stop spack from using nvhpc to build
+  # basic dependencies, some of which don't compile with nvc etc.
+  # Explicitly excluded as modules.
+  - autoconf%gcc
+  - automake%gcc
+  - ca-certificates-mozilla%gcc
+  - diffutils%gcc
+  - gnuconfig%gcc
+  - libiconv%gcc
+  - libxcrypt%gcc
+  - libxml2%gcc
+  - m4%gcc
+  - ncurses%gcc
+  - openssl%gcc
+  - perl%gcc
+  - xz%gcc
+  - zlib%gcc
+  - zstd%gcc
+  - c-blosc%gcc
+  - libaec%gcc
+  - jasper%gcc
+  - patchelf%gcc
+  - gmake%gcc
+  - paraview@5.12%gcc +mpi +tbb +egl ~qt +fortran +python +hdf5 +nvindex +cdi +raytracing +libcatalyst
+  - llvm~cuda
+  - libcatalyst +fortran +python +mpi
+  variants:
+  - cuda_arch=90
+  - +mpi
+  - +cuda
+  packages:
+  - curl
+  - gmake
+  views:
+    default:
+      link: run
```

Comment:
This has been changed to `run` so that all dependencies are exposed in the view.

In particular, without this paraview would not get all dependencies in the view, e.g. `mpi4py` python package would not be exposed (while it should).

Reply:
@bcumming apparently you are the author of the ICON image from where I started.

Do you think this change is problematic? I can try thinking about alternatives, but I don't have the full view of the thing (e.g. should we have a separate view just for paraview?!)

Reply:
`link: run` is problematic, because it breaks many command line utilities because they will find dependencies in `/user-environment/env/$envname/lib{64}`, when they were linked against the versions in `/user/lib` etc.

The preferred approach is to add all of the specs that you need to expose as roots.

Reply:
we use python, and netcdf from the ICON image (and perhaps other things I forgot). I don't understand enough about the setup to tell if things must be re-build indipendently. But for sure, the ICON-ParaView interface would only be used by ICON users, while ParaView will be used by multiple communities. ParaView must cater to many use-cases (other netcdf users without ICON) (in-situ is another example where the need to have mpi4py exposed forced us to use `link: run`.

Reply:
Because Paraview should be used by multiple communities, it makes sense to make a separate paraview uenv.
The image can provide its own `netcdf` etc.